### DISCUSSION TITLE
Character turn around slider in scene!

### DISCUSSION THREAD

**Discussion by TempleLionStudios** [keiwando/toonsquid#686] [Ideas]
Adobe Animate showcased how their AI can generate turn arounds for your character. This is next level and I don't expect something like this for TS, but how about we draw the frames of the turn around our selves and have a slider above the character, that spins the character according to the frames we've drawn? 

I know markers already does this, but going into the menu, then markers, then selecting the correct frame is a bit counter-intuitive and time consuming (every second of work counts in animation!) 

Also an expression window with controls for eyes, mounth, brows and so on would be sooo nice! A detachable window that can float next to the scene.

**Comment by Jebs88** [comment]
This. Hoping for something close to Moho App.

**Reply by TempleLionStudios** [reply]
> This. Hoping for something close to Moho App.

That would be really awesome! Man, that would be revolutionary to have on an ipad for sure!
